src/main/resources/sectiontool.properties

```diff
@@ -51,6 +51,14 @@ canvas.api.search.courses.regex=^/api/v1/accounts/1/courses\\?search_term=.*&per
 #/api/v1/accounts/self/users?search_term=instx_='+ generateCurrentTimestamp();
 canvas.api.search.user.regex = ^/api/v1/accounts/self/users\\?search_term=.*$
 
+#POST
+#/api/v1/accounts/1/users?account_id=1&user[name]=Gsilver3%20Test19&pseudonym[unique_id]=gsilver3-test19+mailinator.com&pseudonym[sis_user_id]=gsilver3-test19@mailinator.com&pseudonym[send_confirmation]=false&communication_channel[type]=email&communication_channel[address]=gsilver3-test19@mailinator.com&communication_channel[skip_confirmation]=false
+canvas.api.create.user.regex=^/api/v1/accounts/1/users\\?account_id=1&user\\[name\\]=.*&pseudonym\\[unique_id\\]=.*&pseudonym\\[sis_user_id\\]=.*&pseudonym\\[send_confirmation\\]=.*&communication_channel\\[type\\]=.*&communication_channel\\[address\\]=.*&communication_channel\\[skip_confirmation\\]=.*$
```

Comment:
I was close to drawing that conclusion myself.  I think the code would be easier to maintain if it were written to execute a specific piece of logic based on the beginning of the URL or a certain parameter.  That code would then validate whether all the required parameters are present and their values are of the correct types.  If the validation fails, then an error could be returned or the code could fall through to other logic, etc.

Not to be misunderstood: I *love* regexes, but I think it's overkill in this case.